client/src/components/Popup.tsx

```diff
@@ -27,14 +45,22 @@ const Popup = (props: PopupProps) => {
         <>
             <div
                 className="fixed left-0 top-0 z-20 w-screen h-screen bg-black/30"
-                onClick={() => props.setEnabled(false)}
+                onClick={() => setEnabled(false)}
             ></div>
             <div
                 className={twMerge(
                     'fixed z-30 left-1/2 top-1/2 translate-x-[-50%] translate-y-[-50%] px-6 py-6 md:px-10 md:w-1/3 w-11/12 rounded-md border-2 border-lighter border-solid flex flex-col bg-background',
                     props.className
                 )}
             >
+                {/* x to close the popup */}
+                <button
+                    onClick={() => setEnabled(false)}
+                    className="absolute top-2 left-2 text-gray-500 hover:text-gray-700 text-2xl"
+                    aria-label="Close popup"
+                >
+                    &times;
+                </button>
```

Comment:
![image](https://github.com/user-attachments/assets/23b24fb7-be6e-4c80-9dbc-7c9c3c1bc795)


Can the X be more evenly placed (same dist from left and top) and then have it be a little bit bigger and use `lightest` color.

Reply:
Also add a hover color -- whatever looks good (prob black or `light` color)